The derby.properties is not currently being installed -- defaults
seem to work fine...

--- library/derby.properties	2010-08-06 17:18:38.000000000 -0400
+++ library/derby.properties	2011-01-03 01:53:46.000000000 -0500
@@ -1,3 +1,2 @@
-derby.stream.error.file=../log/derby.log
 derby.storage.pageSize=8192
 derby.storage.pageCacheSize=3000

The port makes sure, the below file is only used to configure logging of
serviio-daemon process. The console is configured separately -- by
default it simply logs everything to stderr.

--- config/log4j.xml	2011-01-23 14:24:59.000000000 -0500
+++ config/log4j.xml	2011-02-04 00:13:33.000000000 -0500
@@ -1,3 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="US-ASCII"?>
 <!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
 
@@ -10,8 +10,8 @@
 <log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/" debug="false">
 
 	 <appender name="FILE" class="org.apache.log4j.RollingFileAppender">
 	 	<param name="Threshold" value="DEBUG"/>
 	 	<param name="Append" value="true" />
-		<param name="File" value="${serviio.home}/log/serviio.log" />
+		<param name="File" value="/var/log/serviio/serviio.log" />
 		<param name="MaxFileSize" value="500KB" />
 		<param name="MaxBackupIndex" value="5" />	
